src/pages/proposal.tsx

```diff
@@ -262,86 +262,88 @@ export default function ProposalPage() {
+                    currentUpdate={currentUpdate}
+                    pendingUpdates={pendingUpdates}
+                    onPostUpdateClick={handlePostUpdateClick}
+                  />
+                )}
+                <ProposalResultSection
+                  disabled={!proposal || !votes}
+                  loading={voting || proposalState.loading || votesState.loading || votingPowerState.loading}
+                  proposal={proposal}
+                  votes={votes}
+                  votingPower={votingPower || 0}
+                  changingVote={options.changing}
+                  onChangeVote={(_, changing) => patchOptions({ changing })}
+                  onOpenVotesList={() => patchOptions({ showVotesList: true })}
+                  onVote={(_, choice, choiceIndex) => vote(choice, choiceIndex)}
+                />
+                <ProposalDetailSection proposal={proposal} profile={profile} />
+                {(isOwner || isCommittee) && (
                   <Button
                     basic
-                    loading={updatingStatus}
+                    loading={deleting}
                     style={{ width: '100%' }}
-                    onClick={() => patchOptions({ confirmStatusUpdate: ProposalStatus.Passed })}
+                    disabled={proposal?.status !== ProposalStatus.Pending && proposal?.status !== ProposalStatus.Active}
+                    onClick={() => patchOptions({ confirmDeletion: true })}
                   >
-                    {t('page.proposal_detail.pass')}
+                    {t('page.proposal_detail.delete')}
                   </Button>
+                )}
+                {isCommittee && proposal?.status === ProposalStatus.Passed && (
                   <Button
                     basic
                     loading={updatingStatus}
                     style={{ width: '100%' }}
                     onClick={() =>
                       patchOptions({
-                        confirmStatusUpdate: ProposalStatus.Rejected,
+                        confirmStatusUpdate: ProposalStatus.Enacted,
                       })
                     }
                   >
-                    {t('page.proposal_detail.reject')}
+                    {t('page.proposal_detail.enact')}
                   </Button>
-                </>
-              )}
+                )}
+                {isCommittee && proposal?.status === ProposalStatus.Finished && (
+                  <>
+                    <Button
+                      basic
+                      loading={updatingStatus}
+                      style={{ width: '100%' }}
+                      onClick={() => patchOptions({ confirmStatusUpdate: ProposalStatus.Passed })}
+                    >
+                      {t('page.proposal_detail.pass')}
+                    </Button>
+                    <Button
+                      basic
+                      loading={updatingStatus}
+                      style={{ width: '100%' }}
```

Comment:
I think it's a good opportunity to move these `style={{ width: '100%' }}` to a css class, what do you think? There are a bunch on this page 😬

Reply:
Maybe we can create an issue to remove inline styles

Reply:
Fixed in #310